target/consensource
# Add mock factory data to genesis block

Signed-off-by: Patrick-Erichsen <Patrick.Erichsen@target.com>

## Proposed change/fix

Add two mock factories to the genesis block and rename the CLI from `creg` -> `csrc`.

## Types of changes

What types of changes is this pull request introducing to ConsenSource? _Put an `x` in the boxes that apply_


## How to run/test

1. `docker-compose up`
2. Visit `http://localhost:8080/` and you should see the two mock factories.
